dev/io.openliberty.mcp/bnd.bnd

```diff
@@ -14,7 +14,10 @@ Bundle-SymbolicName: io.openliberty.mcp
 Bundle-Description: Model Context Protocol API, version ${bVersion}
 
 bVersion=1.0
-publish.wlp.jar.suffix: dev/api/ibm
+
+# TODO:  When moving to GA, put this change back in.  We do not want to publish this API to
+# maven until the feature is GA.
+# publish.wlp.jar.suffix: dev/api/ibm
```

Comment:
This is awkward.

I thought the API should be available in the dev directory for people using the beta. I'm not sure how they're supposed to find the APIs if they're not there.

Obviously if this causes them to be published on maven central then we can't do that.